Sming/SmingCore/Network/MqttClient.h

```diff
@@ -316,9 +322,9 @@ class MqttClient : protected TcpClient
 
 #ifndef MQTT_NO_COMPAT
 	// @deprecated
-	MqttMessageDeliveredCallback onDelivery = 0;
+	MqttMessageDeliveredCallback onDelivery = nullptr;
```

Comment:
Since these are the old-style C++ delegates I think that using nullptr instead of 0 will not work as expected. I seem to recall that we already had such an issue but I am not 100% certain.

Reply:
Should  match `Delegate(FunctionDeclaration m)` constructor as `FunctionDeclaration` is a pointer type `ReturnType (*FunctionDeclaration)(ParamsList...)`. Can't see how else it could be interpreted?